apps/web/app/search/page.tsx

```diff
@@ -0,0 +1,83 @@
+'use client';
+
+import RenderProducts from 'components/search/renderProducts';
+import RenderReviews from 'components/search/renderReviews';
+import { Tabs } from 'components/search/tab';
+import { SEARCH_OPTION } from 'constants/option';
+import { SearchOption } from 'types/option';
+import { useState } from 'react';
+import { useSearchParams } from 'next/navigation';
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  BreadcrumbList,
+  BreadcrumbSeparator,
+} from '@/components';
+import { SvgHomeFill } from '@/icons';
+import {
+  mockImageReviewSearchResponse,
+  mockProductSearchResponse,
+  mockVideoReviewSearchResponse,
+} from './mockupData';
+
+export default function Page() {
+  const searchParams = useSearchParams();
+  const middleCategory = searchParams.get('middleCategory');
+  const subCategory = searchParams.get('subCategory');
+  const [selectedTab, setSelectedTab] = useState<SearchOption>(
+    SEARCH_OPTION.PRODUCT
+  );
+
+  const productData = mockProductSearchResponse;
+  const reviewVideoData = mockVideoReviewSearchResponse;
+  const reviewImageData = mockImageReviewSearchResponse;
+
+  const handleClickTab = (option: SearchOption) => {
+    setSelectedTab(option);
+  };
+  const handleVideoButton = () => {};
+  const handleImageButton = () => {};
+
+  const TAB_RENDER = {
+    [SEARCH_OPTION.PRODUCT]: <RenderProducts products={productData.products} />,
+    [SEARCH_OPTION.REVIEW]: (
+      <RenderReviews
+        reviewsVideo={reviewVideoData.reviews}
+        reviewsImage={reviewImageData.reviews}
+        handleVideoButton={handleVideoButton}
+        handleImageButton={handleImageButton}
+      />
+    ),
+  };
```

Comment:
```suggestion
  const TAB_RENDER = {
    [SEARCH_OPTION.PRODUCT]: <RenderProducts products={productData.products} />,
    [SEARCH_OPTION.REVIEW]: (
      <RenderReviews
        reviewsVideo={reviewVideoData.reviews}
        reviewsImage={reviewImageData.reviews}
        handleVideoButton={handleVideoButton}
        handleImageButton={handleImageButton}
      />
    ),
  }[selectedTab];
```
요렇게 하고 아래서 불러오기만 하면 더 간단해보일듯!